src/components/mixins/TransactionMixin.ts

```diff
@@ -27,15 +27,15 @@ export default class TransactionMixin extends Mixins(TranslationMixin, LoadingMi
     if (value.type === Operation.Transfer) {
       params.address = formatAddress(value.to as string, 10)
     }
-    if ([Operation.Transfer, Operation.RemoveLiquidity].includes(value.type)) {
+    if ([Operation.AddLiquidity, Operation.CreatePair, Operation.Transfer, Operation.RemoveLiquidity, Operation.Swap].includes(value.type)) {
       params.amount = params.amount ? this.formatStringValue(params.amount) : ''
     }
-    if ([Operation.AddLiquidity, Operation.CreatePair, Operation.Swap].includes(value.type)) {
+    if ([Operation.AddLiquidity, Operation.CreatePair, Operation.RemoveLiquidity, Operation.Swap].includes(value.type)) {
```

Comment:
pls remove `Operation.RemoveLiquidity` here

Reply:
I added it because I need it for numbers formatting in messages.